### PR TITLE
two little but handy features for MISP v3 integrations

### PR DESCRIPTION
**Pull request by kirion**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- Added *check_to_ids* configuration parameter to respect to_ids flag when doing a lookup in MISP
This is to enable checking of the boolean flag to_ids. The flag allows you to indicate if an attribute should be actionable or not

- Added *allowed_orgs* configuration parameter to filter out results from orgs not on the list
When pulling data from MISP you may have data coming from a reliable source. However, some of the orgs are including things like 8.8.8.8 in their feeds. This parameter is used to filter such orgs out by specifying the exact list of orgs to trust

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by RosenbergYehuda — COMMENTED** [review]
Hello @kirion!

I'd like to begin reviewing your contribution, but I've noticed that you haven't completed the CLA form. Kindly take a moment to fill it out, and I'll be able to proceed with the review.

You can locate the form in previous commits. If you require any assistance, please don't hesitate to reach out.

Have a nice one:)

**Comment by kirion** [comment]
Hi @RosenbergYehuda, I have not used this account for a while and there was a GitHub profile configuration issue which is now fixed. Please advise if any other steps needed from me. 

Thanks!

**Review by RosenbergYehuda — COMMENTED** [review]
Hello @kirion,

I appreciate your prompt response and for taking the time to fill out the missing form. Thank you!

I've begun the code review process, and I must admit that I'm finding it a bit challenging to fully grasp the use case of this change and its general utility.

Would it be possible to schedule a Zoom meeting to discuss this further? It would greatly assist me in comprehending the intricacies before I proceed with my code review.

Please let me know your availability, and we can arrange a suitable time. You can reach me on Slack, respond here, or send me an email, yrosenberg@paloaltonetworks.com .

I hope you have a great weekend!

File: Packs/MISP/Integrations/MISPV3/MISPV3.yml [hunk 1]
```diff
@@ -13,6 +13,21 @@ configuration:
...
+  required: false
+  additionalinfo: This is to enable checking of the boolean flag to_ids. The flag
+    allows you to indicate if an attribute should be actionable or not
+- display: ORG names to use for reputation checks
+  name: allowed_orgs
+  type: 0
+  required: false
+  additionalinfo: Comma-separated list of allowed TI providers (orgc in MISP events). 
+    When pulling data from MISP you may have data coming from a reliable source.
+    However, some of the orgs are including things like 8.8.8.8 in their feeds.
```
Comment (RosenbergYehuda) [inline Packs/MISP/Integrations/MISPV3/MISPV3.yml:29]: if this is a comma separated list, you need to add:
IsArray: true

File: Packs/MISP/Integrations/MISPV3/MISPV3.py [hunk 2]
```diff
@@ -41,6 +41,8 @@ def warn(*args):
 if not params.get('credentials') or not (MISP_API_KEY := params.get('credentials', {}).get('password')):
     raise DemistoException('Missing API Key. Fill in a valid key in the integration configuration.')
 MISP_URL = params.get('url')
+TO_IDS = params.get('check_to_ids')
+ALLOWED_ORGS = argToList(params.get('allowed_orgs'), ',')
```
Comment (RosenbergYehuda) [inline Packs/MISP/Integrations/MISPV3/MISPV3.py:45]: I just want to verify, if the user did not enter any ORGS, the default value will be ",".

Is that equal to any?

**Review by RosenbergYehuda — COMMENTED** [review]
Nice job Yury!

I added 2 minor comments. Please take a look when you have time.

Since you already showed me the way it works, I think we don't need another live demo.
Let met know whenyou'rer done.

**Comment by RosenbergYehuda** [comment]
Hello @kirion,
I want to mention that due to the holiday season in Israel, our team will be out of the office until next Tuesday the 26th. Please feel free to address the comments at your convenience, and I will review again when I'm back.

Thank you again for your contribution!

**Comment by RosenbergYehuda** [comment]
Hello @kirion ,
I wanted to let you know that I will be on vacation next week due to the holidays here in Israel. I will return to work on January 8th.

Please accept my apologies for any delays this may cause. 
Looking forward to catching up when I'm back in the office. 

Regards,Yehuda

**Comment by RosenbergYehuda** [comment]
My mistake, I typed January instead of October, I am on vacation until the 8th of October.

**Comment by RosenbergYehuda** [comment]
Hi @kirion,

I'm still waiting for your response to the comments I provided above. Please respond so we can move forward.

**Comment by kirion** [comment]
> Hi @kirion,
> 
> I'm still waiting for your response to the comments I provided above. Please respond so we can move forward.

Hi @RosenbergYehuda , I have left my comments last week and pushed another commit as well. Please review

**Comment by RosenbergYehuda** [comment]
Hi @kirion,

I don't see any responses to my comments. Would you like me to set up a meeting so we can discuss all the things that are unclear to me?

File: Packs/MISP/Integrations/MISPV3/MISPV3.py (hunk 2 above)
Comment (kirion) [reply Packs/MISP/Integrations/MISPV3/MISPV3.py:45]: @RosenbergYehuda

File: Packs/MISP/Integrations/MISPV3/MISPV3.yml (hunk 1 above)
Comment (kirion) [reply Packs/MISP/Integrations/MISPV3/MISPV3.yml:29]: @RosenbergYehuda see the changes. I followed the way how it is defined for other parameters

**Comment by kirion** [comment]
I forgot to submit the review :) now it should be ok

File: Packs/MISP/Integrations/MISPV3/MISPV3.yml (hunk 1 above)
Comment (RosenbergYehuda) [reply Packs/MISP/Integrations/MISPV3/MISPV3.yml:29]: You are right, I did not realize that "Malicious tag IDs" and "Suspicious tag IDs" are type 12.

**Review by RosenbergYehuda — APPROVED** [review]

**Pull request merged**